medinify/scrapers/webmd_scraper.py

```diff
@@ -48,7 +49,7 @@ def scrape(self, input_url, pages):
         """
 
         quote_page1 = input_url + '&pageIndex='
-        quote_page2 = '&sortby=3&conditionFilter=-500'
+        quote_page2 = '&sortby=3&conditionFilter=-1'
```

Comment:
Why does this need to be -1?

Reply:
We realized it needed to be -1 when we trying to scrap from webMD because it wasn't scraping all the reviews